v0.4.0/Azure-NDv4-A100-quickscan/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
pls check https://docs.opensource.microsoft.com/releasing/maintain/license/ for latest version

Reply:
thanks, updated to the latest